### PR TITLE
Services migration

### PR DESCRIPTION
**Pull request by GuillaumeDSM**
requires https://github.com/Drakkar-Software/OctoBot-Tentacles/pull/91

### CODE REVIEW

File: Interfaces/services/__init__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#  Drakkar-Software OctoBot-Tentacles
```
Comment (Herklos) [inline Interfaces/services/__init__.py]: https://github.com/Drakkar-Software/OctoBot-Tentacles/pull/89#discussion_r362510407

File: Interfaces/services/twitter_service.py [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+            return await self.split_tweet_content(content=content, tweet_id=tweet_id)
+        except Exception as e:
+            error = f"Failed to send response tweet : {e} tweet:{content}"
+            if error_on_failure:
+                self.logger.error(error)
+            else:
+                self.logger.info(error)
+        return None
+
+    async def split_tweet_content(self, content, counter=None, counter_max=None, tweet_id=None):
```
Comment (Herklos) [inline Interfaces/services/twitter_service.py]: I don't understand why this method has to be async ?

Comment (GuillaumeDSM) [reply Interfaces/services/twitter_service.py]: legacy issue, I'll fix it and rename the method at the same time, it splits and sends, not just split

Comment (Herklos) [reply Interfaces/services/twitter_service.py]: Nice :+1:

**Pull request merged**